include/core/parameters_lagrangian.h

```diff
@@ -256,6 +256,21 @@ namespace Parameters
       // Center of mass
       Point<dim> point_center_mass;
 
+      // Boundary Inertia
+      Tensor<1,3> boundary_inertia;
```

Comment:
What is boundary inertia? Is it the moment of inertia of the solid triangulation?

Reply:
I changed to triangulation_inertia